SU2_DEF/obj/Makefile.am

```diff
@@ -48,28 +48,9 @@ ___bin_SU2_DEF_SOURCES = \
   
 
 ___bin_SU2_DEF_CXXFLAGS =
-___bin_SU2_DEF_LDADD = ../../Common/lib/libSU2.a \
-  ../../SU2_CFD/src/libSU2Core_a-output_structure.o \
-  ../../SU2_CFD/src/libSU2Core_a-output_cgns.o \
-  ../../SU2_CFD/src/libSU2Core_a-output_tecplot.o \
-  ../../SU2_CFD/src/libSU2Core_a-output_fieldview.o \
-  ../../SU2_CFD/src/libSU2Core_a-output_su2.o \
-  ../../SU2_CFD/src/libSU2Core_a-output_paraview.o \
-  ../../SU2_CFD/src/libSU2Core_a-solver_structure.o \
-  ../../SU2_CFD/src/libSU2Core_a-variable_structure.o \
-  ../../Common/src/toolboxes/MMS/libSU2_a-CVerificationSolution.o \
-  ../../Common/src/toolboxes/MMS/libSU2_a-CIncTGVSolution.o \
-  ../../Common/src/toolboxes/MMS/libSU2_a-CInviscidVortexSolution.o \
-  ../../Common/src/toolboxes/MMS/libSU2_a-CMMSIncEulerSolution.o \
-  ../../Common/src/toolboxes/MMS/libSU2_a-CMMSIncNSSolution.o \
-  ../../Common/src/toolboxes/MMS/libSU2_a-CMMSNSTwoHalfCirclesSolution.o \
-  ../../Common/src/toolboxes/MMS/libSU2_a-CMMSNSTwoHalfSpheresSolution.o \
-  ../../Common/src/toolboxes/MMS/libSU2_a-CMMSNSUnitQuadSolution.o \
-  ../../Common/src/toolboxes/MMS/libSU2_a-CMMSNSUnitQuadSolutionWallBC.o \
-  ../../Common/src/toolboxes/MMS/libSU2_a-CNSUnitQuadSolution.o \
-  ../../Common/src/toolboxes/MMS/libSU2_a-CRinglebSolution.o \
-  ../../Common/src/toolboxes/MMS/libSU2_a-CTGVSolution.o \
-  ../../Common/src/toolboxes/MMS/libSU2_a-CUserDefinedSolution.o
+___bin_SU2_DEF_LDADD = \
+  ../../SU2_CFD/obj/libSU2Core.a \
+  ../../Common/lib/libSU2.a
```

Comment:
Is there any reason to link against individual object files instead of the static library?

Reply:
Actually there is no reason :+1: Thanks!

Reply:
agreed.. I think that's just legacy behavior from before the libSU2Core.a was added around the time of the in-memory python wrapper